ogcapi-features/extensions/transaction/README.md

```diff
@@ -40,47 +40,54 @@ work to get it incorporated.
 
 When the body is a partial Item:
 
-- Shall only create a new resource.
-- Shall return 409 if an Item exists for the same collection and id values.
-- Shall populate the `collection` field in the Item from the URI.
-- Shall return 201 and a Location header with the URI of the newly added resource for a successful operation.
+- Must only create a new resource.
+- Must have an id field.
+- Must return 409 if an Item exists for the same collection and id field values.
+- Must populate the `collection` field in the Item from the URI.
+- Must return 201 and a Location header with the URI of the newly added resource for a successful operation.
 - May return the content of the newly added resource for a successful operation.
 
 When the body is a partial ItemCollection:
 
-- Shall only create a new resource.
-- Shall return 409 if an Item exists for any of the same collection and id values.
-- Shall populate the `collection` field in each Item from the URI.
-- Shall return 201 without a Location header.
-- May create only some of the Items in the ItemCollection.
+- Must only create a new resource.
+- Each Item in the ItemCollection must have an id field.
+- Must return 409 if an Item exists for any of the same collection and id values.
+- Must populate the `collection` field in each Item from the URI.
+- Must return 201 without a Location header.
+- May create only some of the Items in the ItemCollection. Implementations are not
+  required to implement all-or-none sematics for this operation. For example, if an
+  ItemCollection contains two Items and one is successfully created and the other
+  fails to be created, the server is not required to then delete the successfully
+  created one. When only some of the Items in the ItemCollection are created, the
+  server should communicate this failure back to the client with an error status code.
```

Comment:
Should it be defined which error code should be used when only a partial update happened?

Reply:
I don't think we can. It could be any problem. I think the point here is that it's not atomic, and you may have partial updates, and that the server isn't required to enforce atomicity.